pkg/ovs/ovn-nbctl.go

```diff
@@ -427,6 +418,10 @@ func (c Client) DeleteLogicalSwitch(ls string) error {
 // CreateLogicalRouter delete logical router in ovn
 func (c Client) CreateLogicalRouter(lr string) error {
 	_, err := c.ovnNbCommand(MayExist, "lr-add", lr)
+	if err == nil {
```

Comment:
combine `lr-add` and `set` in one command

Reply:
done.
add agrs to list entities.

Reply:
I can't find all git logs in my record...
may need help with squash and merge.